tastytaps/taps/admin.py

```diff
@@ -1,3 +1,6 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import Price
+
+
+admin.site.register(Price)
```

Comment:
We could leave it until all the APIs are in place. Maybe just add a `#TODO` so we know the intention.